tools/flowc/tests/js/wabt1/readme.md

```diff
@@ -0,0 +1,24 @@
+Running the test
+----------------
+
+To run the test:
+
+    flowc1 nodejs=1 es6=test.js readable=1 wabt_test.flow
```

Comment:
Can we get it to work without "readable=1"?

Reply:
To make it working without 'readable=1' we need to access 'HaxeRuntime._structids_' in WabtNativeHost.ts - and yet I don't know how to do it.